agent/bench-scripts/tests/pbench-fio/test-52.txt

```diff
@@ -0,0 +1,460 @@
++++ Running test-52 pbench-fio
```

Comment:
Let's just punt this test entirely.

Ideally, this would be some functional test that is known to take a "long" time.

But we really want the unit tests to finish somewhat quickly.

Future work is going to move all this to the server side in Python3 anyways, so we'll want to solve such functional verification requirements other ways

Reply:
With the changes in this PR, this test no longer takes very long to run, but it does nearly double the length of time required to run unit tests.  On that basis, plus the fact that it's not a "unit test", I'll pull it from this PR.

However, we should keep it and run it -- it's a valid and valuable functional test.  Where should I put it and how do I hook it up?

Reply:
Is PR #2365 where this landed?

Reply:
> Is PR #2365 where this landed?

Yup.